github3/events.py

```diff
@@ -14,7 +14,7 @@
 class Event(GitHubCore):
 
     """The :class:`Event <Event>` object. It structures and handles the data
-    returned by via the `Events <http://developer.github.com/v3/events>`_
+    returned by via the `Events <https://developer.github.com/v3/activity/events>`_
```

Comment:
Can you break after Events here to fix https://travis-ci.org/sigmavirus24/github3.py/jobs/136800116#L166 ?